interfaces/builtin/qualcomm_ipc_router.go

```diff
@@ -40,32 +45,221 @@ const qipcrtrBaseDeclarationSlots = `
   qualcomm-ipc-router:
     allow-installation:
       slot-snap-type:
+        - app
         - core
-    deny-auto-connection: true
+    allow-connection:
+      plug-attributes:
+        qcipc: $SLOT(qcipc)
```

Comment:
we can maybe do this to solve that problem:

```
allow-connection:
   - plug-attributes:
       qcipc: $SLOT(qcipc)
   - plug-attributes:
       qcipc: $MISSING
     slot-attributes:
       qcipc: $MISSING
```
which means either both side have the same qcipc or both sides don't have it